bundles/org.openhab.binding.fronius/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -43,14 +43,24 @@
 			</channel>
 			<channel id="powerflowautonomy" typeId="powerflow_rel_autonomy"/>
 			<channel id="powerflowselfconsumption" typeId="powerflow_rel_selfconsumption"/>
+			<channel id="powerflowinverterpower" typeId="inverterPower">
+				<label>Inverter Power</label>
+			</channel>
+			<channel id="powerflowinvertersoc" typeId="inverterSoc">
+				<label>State of Charge</label>
```

Comment:
Why redefining a label and not using the one from the channel type ?
Once removed here, the entry in the i18n file should also be removed.